### PR TITLE
 chore: bump apisix to 2.7-alpine

### PR DESCRIPTION
**Pull request by anjia0532**

### CODE REVIEW

**Comment by tokers** [comment]
@anjia0532 Please also change the app version.

**Comment by anjia0532** [comment]
@tokers done

File: charts/apisix/Chart.yaml
Comment (tokers) [inline charts/apisix/Chart.yaml:47]: The apisix-dashboard chart `0.1.5` hasn't released yet. This change will break the helm dependencies installation.

**Comment by tokers** [comment]
We should need to wait until #108 is merged.

**Comment by tao12345666333** [comment]
You should run helm dependence update to update all dependence charts.

**Comment by anjia0532** [comment]
@tao12345666333 
```
 sudo helm dependency update  -n apisix  --kubeconfig /etc/rancher/k3s/k3s.yaml
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "rancher-stable" chart repository
...Successfully got an update from the "apisix" chart repository
...Successfully got an update from the "jetstack" chart repository
...Successfully got an update from the "bitnami" chart repository
Update Complete. ⎈Happy Helming!⎈
Error: can't get a valid version for repositories apisix-dashboard. Try changing the version constraint in Chart.yaml

```

**Comment by anjia0532** [comment]
> You should run helm dependence update to update all dependence charts.
reduce apisix-dashboard version to 0.1.4
```
sudo helm dependency update  -n apisix  --kubeconfig /etc/rancher/k3s/k3s.yaml
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "rancher-stable" chart repository
...Successfully got an update from the "apisix" chart repository
...Successfully got an update from the "jetstack" chart repository
...Successfully got an update from the "bitnami" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 3 charts
Downloading etcd from repo https://charts.bitnami.com/bitnami
Downloading apisix-dashboard from repo https://charts.apiseven.com
Downloading apisix-ingress-controller from repo https://charts.apiseven.com
Deleting outdated charts

```

**Comment by tao12345666333** [comment]
> 是不是 #108 合并到master，但是没合并到 https://charts.apiseven.com ?

yep.  I have deployed it. Please update apisix-dashboard chart to v0.1.5. thanks!

```
➜  ~ helm search  repo apisix
NAME                                    CHART VERSION   APP VERSION     DESCRIPTION                                    
apisix/apisix                           0.3.4           2.6.0           A Helm chart for Apache APISIX                 
apisix/apisix-dashboard                 0.1.5           2.7.0           A Helm chart for Apache APISIX Dashboard       
apisix/apisix-ingress-controller        0.5.0           1.0.0           Apache APISIX Ingress Controller for Kubernetes
```

**Comment by anjia0532** [comment]
@tao12345666333 Done!

**Comment by tao12345666333** [comment]
And you need run ` helm dependency update charts/apisix` in current repo root dir. You will get some outputs like below.

```
➜  apisix-helm-chart git:(master) ✗ helm dependency update charts/apisix
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "apisix" chart repository
...Successfully got an update from the "ingress-nginx" chart repository
...Successfully got an update from the "bitnami" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 3 charts
Downloading etcd from repo https://charts.bitnami.com/bitnami
Downloading apisix-dashboard from repo https://charts.apiseven.com
Downloading apisix-ingress-controller from repo https://charts.apiseven.com
Deleting outdated charts
➜  apisix-helm-chart git:(master) ✗ gst
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   charts/apisix/Chart.lock
        modified:   charts/apisix/Chart.yaml
        deleted:    charts/apisix/charts/apisix-dashboard-0.1.4.tgz
        modified:   charts/apisix/charts/apisix-ingress-controller-0.5.0.tgz

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        charts/apisix/charts/apisix-dashboard-0.1.5.tgz

no changes added to commit (use "git add" and/or "git commit -a")
```

**Comment by anjia0532** [comment]
@tao12345666333 Please take a review again, thanks.

**Comment by tao12345666333** [comment]
Do you forget to submit the `charts/apisix/Chart.yaml` file?  The `dependencies.apisix-dashboard` should update to 0.1.5

```diff
diff --git a/charts/apisix/Chart.yaml b/charts/apisix/Chart.yaml
index 21edd21..77a1047 100644
--- a/charts/apisix/Chart.yaml
+++ b/charts/apisix/Chart.yaml
@@ -44,7 +44,7 @@ dependencies:
     repository: https://charts.bitnami.com/bitnami
     condition: etcd.enabled
   - name: apisix-dashboard
-    version: 0.1.4
+    version: 0.1.5
     repository: https://charts.apiseven.com
     condition: dashboard.enabled
     alias: dashboard
```

**Comment by anjia0532** [comment]
@tao12345666333  😅 Sorry,Please take a review again.

**Review by tao12345666333 — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Comment by tao12345666333** [comment]
```
➜  ingress-nginx git:(master) ✗ helm repo update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "apisix" chart repository
...Successfully got an update from the "ingress-nginx" chart repository
...Successfully got an update from the "rancher-latest" chart repository
...Successfully got an update from the "bitnami" chart repository
Update Complete. ⎈Happy Helming!⎈
➜  ingress-nginx git:(master) ✗ helm  search repo apisix
NAME                                    CHART VERSION   APP VERSION     DESCRIPTION                                    
apisix/apisix                           0.3.5           2.7.0           A Helm chart for Apache APISIX                 
apisix/apisix-dashboard                 0.1.5           2.7.0           A Helm chart for Apache APISIX Dashboard       
apisix/apisix-ingress-controller        0.5.0           1.0.0           Apache APISIX Ingress Controller for Kubernetes
```

**Pull request merged**
